backend/moonstream/routes/users.py

```diff
@@ -149,21 +149,72 @@ async def delete_user_handler(
 
 @app.post("/token", tags=["tokens"], response_model=BugoutToken)
 async def login_handler(
-    username: str = Form(...), password: str = Form(...)
+    username: str = Form(...),
+    password: str = Form(...),
+    token_note: Optional[str] = Form(None),
 ) -> BugoutToken:
     try:
         token: BugoutToken = bc.create_token(
             username=username,
             password=password,
             application_id=MOONSTREAM_APPLICATION_ID,
         )
+        if token_note is not None:
+            token = bc.update_token(token.id, token_note=token_note)
```

Comment:
You can remove this line, just released bugout-python v0.1.18 where can be provided `token_note` due creation.
Just update in requirements.txt `bugout==0.1.18` and in setup.py `"bugout >= 0.1.18",`